src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -374,14 +374,17 @@
       (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
     [community-scroll-page community pending?]))
 
-(defn overview
+(defn internal-overview
   [id]
   (let [id                  (or id (rf/sub [:get-screen-params :community-overview]))
         customization-color (rf/sub [:profile/customization-color])]
+    (rn/use-effect #(rf/dispatch [:communities/update-last-opened-at id]) [id])
```

Comment:
Hi @ibrkhalil, Thank you very much for your PR.
Maybe we can create another event, something like ":communities/navigate-to-community" (similar to `:chat/navigate-to-chat`) and handle this event there. wdyt?

Reply:
I think the only benefit for that would be better naming, Will do.